### PR TITLE
(PC-10325) Web - Submit on enter sur les formulaires d inscriptions

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10325

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/auth/forgottenPassword/ReinitializePassword/ReinitializePassword.tsx [hunk 1]
```diff
@@ -65,6 +65,10 @@ export const ReinitializePassword = () => {
     })
   }
 
+  function onSubmitEditing() {
```
Comment (emmajmn) [inline src/features/auth/forgottenPassword/ReinitializePassword/ReinitializePassword.tsx:68]: il se passe quoi si on est pas sur web ? il y a des undefined

Comment (kopax-polyconseil) [reply src/features/auth/forgottenPassword/ReinitializePassword/ReinitializePassword.tsx:68]: `undefined` sur une props = `default`

**Review by BenStirrup — APPROVED** [review]

File: src/features/auth/signup/SetEmail/SetEmail.tsx [hunk 2]
```diff
@@ -91,6 +91,7 @@ export const SetEmail: FunctionComponent = () => {
               autoFocus={true}
               keyboardType="email-address"
               onChangeText={onChangeEmail}
+              onSubmitEditing={Platform.OS === 'web' ? validateEmail : undefined}
```
Comment (Lucasbeneston) [inline src/features/auth/signup/SetEmail/SetEmail.tsx:94]: Je ne savais pas qu'on pouvait choisir 'web' avec Plateform ! Ça risque d'être réutilisé. Il faudrait peut-être prévoir un mini hook pour ça ?

**Review by Lucasbeneston — APPROVED** [review]
👍

**Pull request merged**